MacOS (arm64): root struct of file 'c' has no member named 'utsname'

### Zig Version

0.11.0-dev.3220+447a30299

### Steps to Reproduce and Observed Behavior

## Reproduce
Run the following program:

```zig
const std = @import("std");

pub fn main() !void {
    const stdout_file = std.io.getStdOut().writer();
    var bw = std.io.bufferedWriter(stdout_file);
    const stdout = bw.writer();

    try stdout.print("{}\n", .{std.os.uname()});

    try bw.flush();
}
```

## Result

```
zig build run
zig build-exe hello-world Debug native: error: the following command failed with 1 compilation errors:
/Users/simon/zig/zig/build/stage3/bin/zig build-exe /Users/simon/zig/hello-world/src/main.zig --cache-dir /Users/simon/zig/hello-world/zig-cache --global-cache-dir /Users/simon/.cache/zig --name hello-world --listen=-
Build Summary: 0/5 steps succeeded; 1 failed (disable with -fno-summary)
run transitive failure
└─ run hello-world transitive failure
   ├─ zig build-exe hello-world Debug native 1 errors
   └─ install transitive failure
      └─ install hello-world transitive failure
         └─ zig build-exe hello-world Debug native (reused)
/Users/simon/zig/zig/build/stage3/lib/zig/std/os.zig:195:27: error: root struct of file 'c' has no member named 'utsname'
pub const utsname = system.utsname;
                    ~~~~~~^~~~~~~~
referenced by:
    uname: /Users/simon/zig/zig/build/stage3/lib/zig/std/os.zig:5745:16
    main: src/main.zig:19:51
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```

### Expected Behavior

Something like this:
```
utsname{ .sysname = Darwin, .nodename = xxx, .release = 5.8.12_1, .version = xxxx .machine = x86_64, .domainname = xxxx }
```

--- a/lib/std/c/darwin.zig
+++ b/lib/std/c/darwin.zig
@@ -4161,3 +4161,15 @@ pub const PTHREAD_STACK_MIN = switch (builtin.cpu.arch) {
     .arm, .aarch64 => 16 * 1024,
     else => 8 * 1024,
 };
+
+pub const _SYS_NAMELEN = 256;
+
+pub const utsname = extern struct {
+    sysname: [255:0]u8,
+    nodename: [255:0]u8,
+    release: [255:0]u8,
+    version: [255:0]u8,
+    machine: [255:0]u8,
+};
+
+pub extern "c" fn uname(u: *utsname) c_int;

--- a/lib/std/c/freebsd.zig
+++ b/lib/std/c/freebsd.zig
@@ -2824,3 +2824,15 @@ pub const PTHREAD_STACK_MIN = switch (builtin.cpu.arch) {
     .x86, .powerpc => 4 * 512,
     else => 4 * 1024,
 };
+
+pub const SYS_NMLN = 256;
+
+pub const utsname = extern struct {
+    sysname: [255:0]u8,
+    nodename: [255:0]u8,
+    release: [255:0]u8,
+    version: [255:0]u8,
+    machine: [255:0]u8,
+};
+
+pub extern "c" fn uname(u: *utsname) c_int;

